src/ol/layer/Vector.js

```diff
@@ -17,6 +17,7 @@ import CanvasVectorLayerRenderer from '../renderer/canvas/VectorLayer.js';
  * property on the layer object; for example, setting `title: 'My Title'` in the
  * options means that `title` is observable, and has get/set accessors.
  *
+ * @extends {BaseVectorLayer<import("../source/Vector.js").default>}
```

Comment:
Why was this and other `@extends` added? Typescript knows that from `class ... extends ...`, and the JSDoc plugin generates `@extends` information from there as well.

Reply:
`@extends` is only needed to set the template type for the class. Without it, Typescript knows the parent class but not the template type

Reply:
Ok, please make sure that the API docs can handle that too.

Reply:
Looks like something is broken in TypeScript too here.

Reply:
It's to give a value to the generic attribute :-)